scripts/release-iso

```diff
@@ -45,7 +45,14 @@ if [ -n "$QUBES_GPG_DOMAIN" ]; then
 else
     GPG=gpg
 fi
-$GPG -asb --output "${ISO_BASE}.iso.asc" "${ISO_BASE}.iso"
+
+if [ "${ISO_BASE}.iso" -nt "${ISO_BASE}.iso.asc" ]; then
+    rm -f "${ISO_BASE}.iso.asc"
+fi
+
+if [ ! -e "${ISO_BASE}.iso.asc" ]; then
+    $GPG -asb --output "${ISO_BASE}.iso.asc" "${ISO_BASE}.iso"
+fi
 printf "ok\n"
 
 printf "Generating digests... "
```

Comment:
When skipping already calculated stuff, digests also could use the same approach.

Reply:
Yes indeed. I needed your approval also for this part.